### PR TITLE
Fix tree-shaking

### LINKED ISSUE #52: Uncaught ReferenceError: process is not defined
#### Repro steps:

Run the following code

```js
// src/main.js
import React from 'react';
import App from 'fusion-react';
import Session, {
  SessionSecretToken,
  SessionCookieNameToken,
} from 'fusion-plugin-jwt';

export default async function start() {
  return new App(<div>hello</div>);
}
```

Open browser console

#### Expected

- no error

#### Actual

This error is thrown: `Uncaught ReferenceError: process is not defined`

### PR DESCRIPTION
**Pull request by lhorie**
Resolves #52

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   "devDependencies": {
     "babel-eslint": "8.2.1",
     "babel-plugin-transform-flow-strip-types": "6.22.0",
-    "create-universal-package": "3.3.0",
+    "create-universal-package": "3.3.1",
```
Comment (dennisgl) [inline package.json]: were issues with `3.3.1` fixed?

**Comment by lhorie** [comment]
My bad, had out-of-date local deps.

File: package.json (hunk 1 above)
Comment (KevinGrandon) [reply package.json]: I believe they were only being triggered when you have a default export from another file. I don't see that pattern here, so we should be ok.

Comment (lhorie) [reply package.json]: We're working around them via explicit `import` / `export` declarations rather than ``export from`

**Comment by lhorie** [comment]
!merge

**Pull request merged**